File: Projects/rosalind/TREE.java
import java.io.*;
import java.util.*;

public class TREE {
	public static void main(String[] args) throws IOException {
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		while (in.hasNextLine()) {
			in.nextLine();
			n--;
		}
		System.out.println(n);
		in.close();
	}
}

File: Projects/rosalind/TREE.kt
package rosalind

import java.io.IOException
import java.util.*

object TREE {
    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        var n = stdin.nextLine().toInt()
        while (stdin.hasNextLine()) {
            stdin.nextLine()
            n--
        }
        println(n)
        stdin.close()
    }
}